README.md

```diff
@@ -168,6 +168,8 @@ Many of these parameters are only used in the `:enable` action.
 - **restart_on_update** - Whether the service should be restarted when
     the run script is updated. Defaults to `true`. Set to `false` if
     the service shouldn't be restarted when the run script is updated.
+- **down** - Set the default state of the runit service to 'down' by creating
```

Comment:
Do you think naming it down could lead to future confusion around action 'down'.  Maybe start_down or start_immediately = false might be less prone to future confusion.